core/modules/views/plugins/views_plugin_display_block.inc

```diff
@@ -70,20 +70,24 @@ class views_plugin_display_block extends views_plugin_display {
    */
   function execute_hook_block_list($delta = 0, $edit = array()) {
     $delta = $this->view->name . '-' . $this->display->id;
-    $info = $this->get_option('block_description');
+    $block_title = $this->get_option('block_description');
```

Comment:
perhaps call this variable `$block_admin_title` to match what is in the views UI. That's what I meant when I was saying that we should rename the variables.